### PR TITLE
Fix worldLink transform method in Visualizer

### PR DESCRIPTION
**Pull request by lrapetti**
Fix https://github.com/robotology/idyntree/issues/836

### CODE REVIEW

File: src/visualization/include/iDynTree/Visualizer.h [hunk 1]
```diff
@@ -585,12 +585,11 @@ class IModelVisualization
      * The obtained transformation matrix can be used to map any homogeneous vector from the
      * model's root link frame to the visualizer world frame.
      */
+    IDYNTREE_DEPRECATED_WITH_MSG("Please use the getWorldLinkTransform(<root link frame index>) method to get the root link transformation matrix")
```
Comment (S-Dafarra) [inline src/visualization/include/iDynTree/Visualizer.h:588]:
```suggestion
    IDYNTREE_DEPRECATED_WITH_MSG("This method is simply returning the identity. If you need the root link transformation matrix, please use the getWorldLinkTransform(<root link frame index>) method.")
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed the `IDYNTREE_USES_ASSIMP` option on Windows if `IDYNTREE_USES_YARP` is also enabled (https://github.com/robotology/idyntree/pull/832).
 - Fixed the ``OBJ`` meshes visualization in the Visualizer library (https://github.com/robotology/idyntree/pull/833).
 ## [3.0.0] - 2020-02-03
+- Fixed `ModelVisualization::getWorldLinkTransform()` method (https://github.com/robotology/idyntree/issues/836)
+- Fixed the `IDYNTREE_USES_IRRLICHT` option when `irrlicht` is installed via vcpkg (https://github.com/robotology/idyntree/pull/806).
```
Comment (traversaro) [inline CHANGELOG.md:15]: iDynTree 3.0.0 has been already released, so any new changelog item needs to go under the `Unreleased` section, that will be come the 3.0.1 release section once we release it. See https://keepachangelog.com/en/1.0.0/#effort for more details.

File: CHANGELOG.md [hunk 3]
```diff
@@ -22,9 +24,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Promoted the functions `computeBoundingBoxFromShape` and `computeBoxVertices` to public in the `idyntree-solid-shapes` library (https://github.com/robotology/idyntree/pull/801).
 - The `idyntree-yarp` and `idyntree-icub` libraries are now header-only, and are always installed even if `IDYNTREE_USES_YARP` or `IDYNTREE_USES_ICUB_MAIN` are set to `OFF`, to simplify deployment of the library. The downstream libraries that want to use them need to find and link `YARP` and `ICUB` CMake packages on their own (https://github.com/robotology/idyntree/pull/807).
 
-### Fixed
-- Fixed the `IDYNTREE_USES_IRRLICHT` option when `irrlicht` is installed via vcpkg (https://github.com/robotology/idyntree/pull/806).
-
+### Deprecated
+- The method `ModelVisualization::getWorldModelTransform()` is deprecated, and will be removed in iDynTree 4.0.
```
Comment (traversaro) [inline CHANGELOG.md:28]: An existing section has been removed here.

**Review by traversaro — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (lrapetti) [reply CHANGELOG.md:15]: Sorry, I missed the `## [3.0.0] - 2020-02-03` line in the changelog and I messed up stuff

File: CHANGELOG.md (hunk 3 above)
Comment (lrapetti) [reply CHANGELOG.md:28]: as https://github.com/robotology/idyntree/pull/838#discussion_r586319294

**Review by traversaro — APPROVED** [review]

**Comment by traversaro** [comment]
@S-Dafarra feel free to approve it if you think it make sense.

**Review by S-Dafarra — APPROVED** [review]

**Comment by traversaro** [comment]
Thanks @lrapetti !

**Comment by lrapetti** [comment]
I think you can delete the original @S-Dafarra branch https://github.com/robotology/idyntree/tree/fix/visualization%2FgetWorldPose that I forked

**Comment by S-Dafarra** [comment]
> I think you can delete the original @S-Dafarra branch https://github.com/robotology/idyntree/tree/fix/visualization%2FgetWorldPose that I forked

Done

**Pull request merged**